### DISCUSSION TITLE
About networking process

### DISCUSSION THREAD

**Discussion by tuopu** [openthread/openthread#7576] [Q&A] [answered]
Hello, expert, I used a new set of hardware and completed the transplantation myself, but I found that compared with other hardware, the networking becomes very slow. The old hardware completes the networking within 2 seconds after startup, while the new hardware system (different CPUs) takes a long time and is not sure. Please teach me if there are any instructions on the networking process and frame structure. I'll find out the reasons for the slow networking. Thank you.  thread version：OPENTHREAD/1.2.1.

**Comment by jwhui** [comment, marked as answer]
A [Thread Primer](https://openthread.io/guides/thread-primer) is available on the OpenThread site.

The [Thread 1.1.1 Specification](https://www.threadgroup.org/support#specifications) is available on the Thread Group site.
